Makefile

```diff
@@ -9,21 +9,29 @@
 
 build := build
 project := project
+PREFIX := /
 
 cmake := cmake
 cmake_flags := 
+cmake_command := $(cmake) -H. $(cmake_flags)
 
 ninja := $(if $(shell which llbuild),llbuild ninja build,ninja)
 ninja_flags := $(if $(shell echo "$$NINJA_JOBS"),-j$(shell echo "$$NINJA_JOBS"),)
+ninja_command := $(ninja) $(ninja_flags)
 
 all:
 	mkdir -p $(build)
-	$(cmake) -B$(build) -H. -G Ninja $(cmake_flags)
-	$(ninja) -C $(build) $(ninja_flags)
+	$(cmake_command) -B$(build) -GNinja
+	$(ninja_command) -C $(build)
 
 project:
 	mkdir -p $(project)
-	$(cmake) -B$(project) -H. -G Xcode $(cmake_flags)
+	$(cmake_command) -B$(project) -GXcode
+
+install:
+	mkdir -p $(PREFIX)
+	$(cmake_command) -B$(build) -GNinja -DCMAKE_INSTALL_PREFIX:PATH=$(realpath $(PREFIX)) -DCMAKE_INSTALL_NAME_DIR:STRING=@executable_path/../lib
```

Comment:
Why are we overriding CMAKE_INSTALL_PREFIX instead of using CMake's default? / is an annoying default; it'll install into /bin and /lib, which I doubt anyone expects.

Doesn't CMAKE_INSTALL_NAME_DIR affect compilation? I don't think the install target triggers compilation.